scripts/common/install_yumpackages.sh

```diff
@@ -116,9 +116,12 @@ yum -y groupinstall "Development Tools"
 
 yum install -y --setopt=tsflags=nodocs centos-release-scl-rh yum-utils
 
-sed -i 's/7/7.6.1810/g; s|^#\s*\(baseurl=http://\)mirror|\1vault|g; /mirrorlist/d' /etc/yum.repos.d/CentOS-SCLo-*.repo
-
-yum install -y --setopt=tsflags=nodocs devtoolset-6-toolchain --nogpgcheck
+if [[ $DTS_VERSION == 6 ]]; then
+    sed -i 's/7/7.6.1810/g; s|^#\s*\(baseurl=http://\)mirror|\1vault|g; /mirrorlist/d' /etc/yum.repos.d/CentOS-SCLo-*.repo
```

Comment:
yum-config-manager from the yum-utils package may be a safer approach to editing the repo files than direct surgery with sed -i ?

Reply:
Unfortunately I had to revert the use of yum-utils some time ago as the mirrors also had to be updated... the failure was sudden and I just reverted quickly in https://github.com/AcademySoftwareFoundation/aswf-docker/commit/7e6981d78a563c09a32407fa54c9f3b2d036ed8c